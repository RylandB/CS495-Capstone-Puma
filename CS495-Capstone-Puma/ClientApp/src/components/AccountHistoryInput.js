import React from 'react'

export class AccountHistoryInput extends React.Component{

    render(){
        return(
            <div>
                <h3>Balance History</h3><br />
                <label>1 Month Ago</label><br />
                <input type="text" name="oneMonth" placeholder="Please enter a value" /><br /><br />

                <label>1 Year Ago</label><br />
                <input type="text" name="twoYear" placeholder="Please enter a value" /><br /><br />

                <label>3 Years Ago</label><br />
                <input type="text" name="threeYear" placeholder="Please enter a value" /><br /><br />

                <label>5 Years Ago</label><br />
                <input type="text" name="fiveYear" placeholder="Please enter a value" /><hr />

                <label>Value added this month</label><br />
                <input type="text" name="valThisMonth" placeholder="Please enter a value" /><br /><br />

                <label>Value added this year</label><br />
                <input type="text" name="valThisYear" placeholder="Please enter a value" /><br /><br />
            </div>
        );
    }

}