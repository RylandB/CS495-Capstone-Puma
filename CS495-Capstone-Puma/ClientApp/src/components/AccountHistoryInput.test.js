import React from "react";
import {AccountHistoryInput} from "./AccountHistoryInput";

it("AccountHistoryInput Renders Correctly", () => {
    const wrapper = shallow(
        <AccountHistoryInput />
    );

    expect(wrapper).toMatchSnapshot();
});