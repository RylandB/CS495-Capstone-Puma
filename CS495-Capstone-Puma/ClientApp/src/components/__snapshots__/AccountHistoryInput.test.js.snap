// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountHistoryInput Renders Correctly 1`] = `
<div>
  <h3>
    Balance History
  </h3>
  <br />
  <label>
    1 Month Ago
  </label>
  <br />
  <input
    name="oneMonth"
    placeholder="Please enter a value"
    type="text"
  />
  <br />
  <br />
  <label>
    1 Year Ago
  </label>
  <br />
  <input
    name="twoYear"
    placeholder="Please enter a value"
    type="text"
  />
  <br />
  <br />
  <label>
    3 Years Ago
  </label>
  <br />
  <input
    name="threeYear"
    placeholder="Please enter a value"
    type="text"
  />
  <br />
  <br />
  <label>
    5 Years Ago
  </label>
  <br />
  <input
    name="fiveYear"
    placeholder="Please enter a value"
    type="text"
  />
  <hr />
  <label>
    Value added this month
  </label>
  <br />
  <input
    name="valThisMonth"
    placeholder="Please enter a value"
    type="text"
  />
  <br />
  <br />
  <label>
    Value added this year
  </label>
  <br />
  <input
    name="valThisYear"
    placeholder="Please enter a value"
    type="text"
  />
  <br />
  <br />
</div>
`;
